app/src/main/res/values/dimens.xml

```diff
@@ -7,5 +7,5 @@
   <dimen name="audio_fragment_margin">32dp</dimen>
   <dimen name="cellular_data_dialog_padding">24dp</dimen>
   <dimen name="state_previous_next_button_radius">4dp</dimen>
-  <dimen name="padding_8">8dp</dimen>
+  <dimen name="padding_8dp">8dp</dimen>
```

Comment:
Prefer adding contextual padding that will specifically be shared. If this is just meant to represent 8dp and isn't significant in some other way (e.g. it's not the padding for all textual interaction views), then please remove this and just use 8dp directly where it's needed.

Reply:
removed